addon_service/external_citation_service/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+from addon_service.abstract.external_storage.models import ExternalService
+
+
+class ExternalCitationService(ExternalService):
+    int_addon_imp = models.IntegerField(
+        null=False,
+        verbose_name="Addon implementation",
+    )
+    # Do we need the wb_key field for ExternalCitationService ?
+    wb_key = models.CharField(null=False, blank=True, default="")
```

Comment:
`wb_key` should not be necessary here since there are no file ops.

Reply:
@jwalz stated that we need `wb_key` inside `ExternalCitationService`, for us to map credentials from `osf` addons to `GV` ones. We don't need wb_key inside `CitationConfig` class. `wb_key` is quite misleading